pallets/collator-assignment/src/lib.rs

```diff
@@ -236,6 +236,14 @@ pub mod pallet {
                     max_collators: collators_per_parathread,
                 });
             }
+
+            // Prioritize paras by tip
+            // This doesn't distinguish between parachains and parathreads
```

Comment:
We should put a TODO here. It's likely we want to treat parathread tips differently in the future (as they have different collator needs)